silabs-multiprotocol/config.yaml

```
@@ -0,0 +1,28 @@
+version: 0.1.0
+slug: silabs_multiprotocol
+name: Silicon Labs Multiprotocol
+description: Zigbee and OpenThread multiprotocol add-on
+url: https://github.com/home-assistant/hassio-addons/tree/master/silabs-multiprotocol
+arch:
+  - armv7
+# Currently zigbeed crashes with *** stack smashing detected ***
+#  - aarch64
+hassio_api: true
+privileged:
+  - IPC_LOCK
+image: ghcr.io/home-assistant/{arch}-addon-silabs-multiprotocol
```

You need to add shadow repository for this <https://github.com/home-assistant/supervisor/blob/main/build.yaml#L2>

---

Oh I see, the other addons are still on Docker Hub then?

I don't see a reason why we should make this special, so how about:

```suggestion
image: homeassistant/{arch}-addon-silabs-multiprotocol
```

---

That works :+1: 
we can also remove the ghcr.io login part of the CI BTW, it's not used so no need to be there

---

Ok, I'll do that in a separate PR. It is also not needed in addons repo afaict.